### PR TITLE
Network generator read/write ratio

### PR DESCRIPTION
**Pull request by abelov-spirent**
Signed-off-by: Artem Belov <artem.belov@spirent.com>

---
This change is [Reviewable](https://reviewable.io/reviews/spirent/openperf/438)

### CODE REVIEW

File: src/modules/network/task.hpp [hunk 1]
```diff
@@ -27,6 +27,14 @@ struct target_t
     std::optional<std::string> interface;
 };
 
+struct synchronizer_t
+{
```
Comment (cmyers-spirent) [inline src/modules/network/task.hpp:31]: I don't see ratio_reads/ratio_writes get modified.
Why is there a need to make them atomic?

Comment (abelov-spirent) [reply src/modules/network/task.hpp:31]: ratio_reads/ratio_writes are changing each worker iteration, and can be read at other thread. So atomic is required here

Comment (abelov-spirent) [reply src/modules/network/task.hpp:31]: It is full copy of such option in block module

**Review by cmyers-spirent — APPROVED** [review]
> Reviewed 19 of 19 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/spirent/openperf/438)* status: all files reviewed, 1 unresolved discussion (waiting on @cmyers-spirent and @DerangedMonkeyNinja)

**Review by cmyers-spirent — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/spirent/openperf/438)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @DerangedMonkeyNinja)

**Pull request merged**